301-multi-vmss-linux/azuredeploy.json

```diff
@@ -0,0 +1,98 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01-preview/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "vmSSPrefix": {
+            "type": "string"
+        },
+        "numberOfVMSS": {
+            "type": "int",
+            "allowedValues": [
+                1,
+                2,
+                3,
+                4,
+                5
+            ]
+        },
+        "instanceCountPerVMSS": {
+            "type": "string",
+            "metadata": {
+                "description": "Number of VM instances per scale set"
```

Comment:
Meant maxValue like in the other template. See this https://github.com/Azure/azure-quickstart-templates/blob/master/201-vmss-linux-jumpbox/azuredeploy.json#L59